admin/doc-requirements.txt

```diff
@@ -6,6 +6,3 @@ Cython
 prettytable
 sphinx-autodoc-typehints == 1.10.3
 typed-ast
-pcpp
--e../src/python-common
-Jinja2
```

Comment:
this change is not relevant and should be dropped. It appears to be coming from this commit:

```
admin: move python-common deps out of doc-requirements.txt

to address the build failure of

python -m pip install --exists-action=w --no-cache-dir -r admin/doc-requirements.txt
ERROR: ../src/python-common is not a valid editable requirement. It should either be a path to a local project or a VCS URL (beginning with svn+, git+, hg+, or bzr+).

Signed-off-by: Kefu Chai <kchai@redhat.com>
```